helm/templates/deployment.yaml

```diff
@@ -75,13 +72,12 @@ spec:
           livenessProbe:
             initialDelaySeconds: {{ int .Values.livenessProbe.initialDelaySeconds }}
             periodSeconds: {{ int .Values.livenessProbe.periodSeconds }}
-            tcpSocket:
-              port: grpc-port
+            exec:
+              command: [ "/bin/grpc_health_probe", "-addr=:{{ .Values.containerHealthProbePort }}", "-connect-timeout=100ms", "-rpc-timeout=150ms" ]
```

Comment:
How is this `/bin/grpc_health_probe` getting bundled?

Reply:
As per https://github.com/grpc-ecosystem/grpc-health-probe#example-grpc-health-checking-on-kubernetes dont we need to add this into our Dockerfile

```
RUN GRPC_HEALTH_PROBE_VERSION=v0.3.1 && \
    wget -qO/bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 && \
    chmod +x /bin/grpc_health_probe
```

Reply:
I think it's coming from https://github.com/hypertrace/docker-java-base/pull/9

Reply:
See the linked PR - hypertrace/docker-java-base#9 

But I'm wondering if we even need that, and can make this more low touch. If I pipe the grpc result in here but leave the existing checks (just converting that socket check to a get - if there's not a story there), wondering if I actually miss out on anything.

Reply:
Sorry. Missed the linked PR hypertrace/docker-java-base#9

Reply:
>  If I pipe the grpc result in here but leave the existing checks (just converting that socket check to a get - if there's not a story there), wondering if I actually miss out on anything.

Good point. In that case the change in base image to include the grpc_health_probe can be avoided.